Sources/Sentry/SentrySystemEventBreadcrumbs.m

```diff
@@ -177,6 +181,58 @@ - (void)initScreenshotObserver
                                                  name:UIApplicationUserDidTakeScreenshotNotification
                                                object:nil];
 }
+
+- (NSNumber *_Nullable)storedTimezoneOffset
```

Comment:
`l`: I would put this method below `initTimezoneObserver` as `initTimezoneObserver` is the entry point and `storedTimezoneOffset` supports `initTimezoneObserver`.